content/teknologi/altinnstudio/altinn-api/app-api/instances/_index.md

```diff
@@ -0,0 +1,148 @@
+
+```http
+GET {basePath}/{instanceOwnerPartyId}/{instanceGuid}
+```
+
+## Create instance
+
+Altinn assigns an unique identifier to all users that wishes to report data. We call this id *instanceOwner.partyId*. 
+If you do not know this, you should provide the official identity number, e.g national identification number for persons or organisation number for organisations,
+and in some case user name. This should be provided as part of the payload to the creation request.
+Altinn will look up this identifier and replace it with the instanceOwner.PartyId. The official identity number will be stored in the instance metadata.
+
+Data elements can be provided as part of the creation request, but can also be uploaded at a later time.
+
+The client specify the instance owner and can set a number of the metadata fields of the instance by attaching the following form:
+
+```json
+{
+    "appId" : "org/app",
+    "instanceOwner": {
+        "personNumber": "12247918309",
+        "organisationNumber": null
+    },
+    "dueBefore": "2019-06-01T12:00:00Z",
+    "visibleAfter": "2019-05-20T00:00:00Z"
+}
+```
+
+Notice that all dates must be expressed in **Utc (Zulu)** time zone and represented according to ISO 8601!
+
+Data elements (files) can be attached to the initial request as a *multipart/form-data* or as *attachments*.
+The name of the parts must correspond to element types defined in the application metadata. 
+
+```http
+POST {basePath}
+```
+
+![Flow chart for instantiation](instantiate-for-an-instance-owner.png "Instantiate for instance owner")
+
+A multipart formdata should contain the instance json document and the data element files of the instance.
+The first part should be *instance* which contains the json template to create an instance from.
+
+The subsequent parts must have a name that correspond to the element types defined in application metadata.
+They may have a filename. Hence the *model1* and *certificate* names correspond to data types defined in the application metadata.
+If more data elements are needed they must be defined in the application metadata.
+
+```http {linenos=false,hl_lines=[5,10,15]}
+Content-Type: multipart/form-data; boundary="abcdefg"
+
+--abcdefg
+Content-Type: application/json; charset=utf-8
+Content-Disposition: form-data; name="instance"
+{ ... }
+
+--abcdefg
+Content-Type: application/xml
+Content-Disposition: form-data; name="model1"
+<xml> ... </xml>
+
+--abcdefg
+Content-Type: application/pdf
+Content-Disposition: form-data; name="certificate"; filename=certificate.pdf
+%PDF-1.4
+%Óëéá
+1 0 obj
+...
+
+--abcdefg--
+```
+
+This call will return the instance metadata document that was created. 
+
+## Complete instance
+
+Endpoint used by Application Owner to mark an instance as completed. Technically this only means that the instance is no longer needed by the Application Owner. Altinn can delete the instance and associated data should the Application user decide to do so.
```

Comment:
Could we link the reasoning behind deletion closer to the complete action? This way it would make more sense that this information is included here.

E.g. When both application owner and instance owner have indicated that they no longer need the instance, through completion or deletion, Altinn will delete the instance and associated data.